hbsir/metadata/schema.yaml

```diff
@@ -34,6 +34,10 @@ common_expenditure_order: &common_expenditure_order
 
 food:
   settings: *expenditure_tables_settings
+  views:
+   - ORIGINAL1363
```

Comment:
The original classification contains both ORIGINAL1363 and ORIGINAL1383, and the labeling will be handled according to the classification code's corresponding year.